astro/src/content/docs/_shared/_recommended-token-storage-options.mdx

```diff
@@ -0,0 +1,21 @@
+---
+---
+* **Cookie**: send the token down to the browser as a `Secure`, `HttpOnly` cookie. If you don't require cross-site cookie sharing, set `SameSite` to `Strict`. Otherwise, set `SameSite` to `Lax`, which will share the cookies in certain situations. Learn more about [`SameSite` settings](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite).
```

Comment:
```suggestion
* <GreenCheck /> **Cookie**: send the token down to the browser as a `Secure`, `HttpOnly` cookie. If you don't require cross-site cookie sharing, set `SameSite` to `Strict`. Otherwise, set `SameSite` to `Lax`, which will share the cookies in certain situations. Learn more about [`SameSite` settings](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite).
```